_copyjavadoc.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+
+OPENSEARCH_DIR=$1
+CSV_FILE=$2
```

Comment:
Add a comment of what this script does.

Many of these are unused.

Reply:
Thanks for point out. should be fixed now.